test/minimum_eigensolvers/evqe/test_evqe_individual.py

```diff
@@ -0,0 +1,297 @@
+            )
+            assert individual.is_valid()
+
+    def test_random_individual_seed(self):
+        individual_1 = EVQEIndividual.random_individual(
+            n_qubits=15,
+            n_layers=10,
+            randomize_parameter_values=False,
+            random_seed=0,
+        )
+        assert individual_1 == EVQEIndividual.random_individual(
+            n_qubits=15,
+            n_layers=10,
+            randomize_parameter_values=False,
+            random_seed=0,
+        )
+
+        individual_2 = EVQEIndividual.random_individual(
+            n_qubits=20,
+            n_layers=8,
+            randomize_parameter_values=True,
+            random_seed=0,
+        )
+        assert individual_2 == EVQEIndividual.random_individual(
+            n_qubits=20,
+            n_layers=8,
+            randomize_parameter_values=True,
+            random_seed=0,
+        )
+
+    @pytest.fixture
+    def individual(self) -> EVQEIndividual:
+        return EVQEIndividual.random_individual(
+            n_qubits=20, n_layers=10, randomize_parameter_values=True, random_seed=0
+        )
+
+    def test_change_parameter_values(self, individual):
+        n_parameters = len(individual.get_parameter_values())
+        new_parameters = (0,) * n_parameters
+
+        new_individual = EVQEIndividual.change_parameter_values(individual=individual, parameter_values=new_parameters)
+
+        assert new_individual.n_qubits == individual.n_qubits
+        assert new_individual.layers == individual.layers
+        assert new_individual.parameter_values == new_parameters
+
+    def test_change_layer_parameter_values(self, individual):
+        layer_to_change = 4
+        n_params_before = sum(layer.n_parameters for layer in individual.layers[0:layer_to_change])
+        layer_n_params = individual.layers[layer_to_change].n_parameters
+
+        new_parameters = (0,) * layer_n_params
+        new_individual = EVQEIndividual.change_layer_parameter_values(
+            individual=individual, layer_id=layer_to_change, parameter_values=new_parameters
+        )
+
+        assert new_individual.n_qubits == individual.n_qubits
+        assert new_individual.layers == individual.layers
+        assert (
+            new_individual.get_parameter_values()[0:n_params_before]
+            == individual.get_parameter_values()[0:n_params_before]
+        )
+        assert (
+            new_individual.get_parameter_values()[n_params_before : n_params_before + layer_n_params] == new_parameters
+        )
+        assert (
+            new_individual.get_parameter_values()[n_params_before + layer_n_params :]
+            == individual.get_parameter_values()[n_params_before + layer_n_params :]
+        )
+
+    def test_add_random_layers(self, individual):
+        with raises(EVQEIndividualException):
+            EVQEIndividual.add_random_layers(
+                individual=individual, n_layers=0, randomize_parameter_values=False, random_seed=0
+            )
```

Comment:
Maybe move this to a separate test case? It's quite different from the positive tests below.